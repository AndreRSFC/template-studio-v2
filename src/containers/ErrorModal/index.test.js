import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { ErrorModal } from './index';


const props = {
  errorName: 'Error',
  errorMessage: 'Error message',
  errorDescription: 'Unexpected error',
  closeErrorModal: () => null,
};

describe('<ErrorModal />', () => {
  describe('on initialization', () => {
    it('renders page correctly', () => {
      const component = shallow(<ErrorModal {...props} />);
      const tree = toJson(component);
      expect(tree).toMatchSnapshot();
    });
  });
});
