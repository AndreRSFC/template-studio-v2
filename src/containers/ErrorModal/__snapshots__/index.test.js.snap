// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorModal /> on initialization renders page correctly 1`] = `
<Modal
  centered={true}
  closeOnDimmerClick={true}
  closeOnDocumentClick={false}
  dimmer={true}
  eventPool="Modal"
  onClose={[Function]}
  open="Unexpected error"
  size="small"
>
  <Header
    content="Error"
  />
  <Styled(ModalContent)>
    Unexpected error
Error message
  </Styled(ModalContent)>
  <ModalActions>
    <Button
      as="button"
      color="blue"
      id="errorCloseBtn"
      onClick={[Function]}
    >
      Close
    </Button>
  </ModalActions>
</Modal>
`;
